base/database/batch.go

```diff
@@ -63,7 +63,7 @@ func BulkInsertChunk(db *gorm.DB, objects interface{}, chunkSize int) []error {
 	}
 
 	if len(allErrors) > 0 {
-		return allErrors
+		return allErrors[0]
```

Comment:
Why do you return only one error if there may be multiple ones?

Reply:
We don't use the errors anywhere in the codebase, we always just propagate the first one.